.github/workflows/builds.yml

```diff
@@ -1221,15 +1221,15 @@ jobs:
                 python3-yaml \
                 zlib1g \
                 zlib1g-dev
-      - name: Install Rust
-        run: curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain ${RUST_VERSION_MIN} -y
```

Comment:
`RUST_VERSION_MIN` is unused and its definition can go away (as in 7) right ?

Reply:
Related note for later.. I know don't RUST_VERSION_KNOWN provides any value.  Its use can probably be replaced with latest stable.  We already have some that pull rustc from packages, that's a better indicator.